src/KiD_driver.jl

```diff
@@ -27,12 +27,27 @@ face_space = Spaces.FaceFiniteDifferenceSpace(space)
 coord = Fields.coordinate_field(space)
 face_coord = Fields.coordinate_field(face_space)
 
-Yc = map(coord -> init_1d_column(FT, params, coord.z), coord)
-w = Geometry.WVector.(ones(FT, face_space)) #TODO - should be changing in time
-Y = Fields.FieldVector(Yc = Yc, w = w)
+# solve the initial value problem for density profile
+ρ_profile = ρ_ivp(params)
+# create the initial condition profiles
+init = map(coord -> init_1d_column(params, ρ_profile, coord.z), coord)
+w = Geometry.WVector.(ones(FT, face_space))
+
+# initialoze state and aux
+# set initial condition
+Y = Fields.FieldVector(; q_tot = init.q_tot)
+aux = Fields.FieldVector(;
```

Comment:
So if I understand correctly, `q_tot` corresponds to the prognostic variable that is updated with rhs terms in the ODE, and `aux` fields are then recomputed as diagnostics?

Reply:
TLDR - Yes.

Ideally we want to have only the necessary variables in our state `Y` because having them there is expensive. 

Things we should precompute in advance to make life easier when computing tendencies (for example current temperature) can go to aux (or cache or whatever we want to call them). This way we will be computing things only once per time step. 

And then ideally diagnostics should be done separately from aux, because we don't need to do them every time step. But until I hook up the NetCDF output and code diagnostics output we can just dump them to aux